coffea/processor/executor.py

```diff
@@ -1,7 +1,10 @@
 import concurrent.futures
+from functools import partial
 import time
-from tqdm.autonotebook import tqdm
 import uproot
+import pickle
+from tqdm.auto import tqdm
```

Comment:
@paulgessinger has made a separate PR to take care of this
@nsmith-  can you remove the tqdm change from this PR and we can go ahead with the other one?